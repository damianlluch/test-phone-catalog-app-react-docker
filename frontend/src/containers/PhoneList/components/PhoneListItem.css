.PhoneListItem {
  text-align: center;
  float: left;
  margin: 20px;
  width: 275px;
  padding: 15px;
  border-radius: 15px;
  background-color: #FFF;
  color: #5F6368;
}

.PhoneListItem:hover {
  cursor: pointer;
  color: #3C4043;
  box-shadow: 8px 8px 6px -6px #BDC1C6;
}

.PhoneListItem img {
  height: 200px;
}
