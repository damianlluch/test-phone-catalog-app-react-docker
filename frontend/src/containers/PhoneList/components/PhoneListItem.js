import React from 'react';
import PropTypes from 'prop-types';
import './PhoneListItem.css';

const PhoneListItem = props => (
  <article className="PhoneListItem">
    <img src={props.image} alt={props.title} />
    <h2>{props.title}</h2>
    <p>{`Diagonal: ${props.diagonal}"`}</p>
    <p>{`Capacity: ${props.capacity}`}</p>
    <p><b>more specs...</b></p>
  </article>
);

PhoneListItem.propTypes = {
  title: PropTypes.string,
  image: PropTypes.string,
  diagonal: PropTypes.string,
  capacity: PropTypes.string,
};

export default PhoneListItem;
