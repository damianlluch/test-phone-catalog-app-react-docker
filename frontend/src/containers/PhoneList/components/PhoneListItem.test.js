import React from 'react';
import renderer from 'react-test-renderer';
import PhoneListItem from './PhoneListItem';

describe('<PhoneListItem/>', () => {
  it('renders by default', () => {
    const tree = renderer.create(<PhoneListItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
