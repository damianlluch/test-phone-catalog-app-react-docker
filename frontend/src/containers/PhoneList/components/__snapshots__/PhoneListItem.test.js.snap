// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PhoneListItem/> renders by default 1`] = `
<article
  className="PhoneListItem"
>
  <img
    alt={undefined}
    src={undefined}
  />
  <h2 />
  <p>
    Diagonal: undefined"
  </p>
  <p>
    Capacity: undefined
  </p>
  <p>
    <b>
      more specs...
    </b>
  </p>
</article>
`;
